### PR TITLE
Bello2021lambdanetworks 1 marcobueso

### PR DESCRIPTION
**Pull request by marcobueso**
Ready for review.
@SShivang -- Hi Shivang, I apologize for turning this summary in so late. If you can take a look at it on Saturday or Sunday, I can make the changes. Otherwise, I talked to our TA Yue Zhao, and they said it's okay if you don't review this one.

### CODE REVIEW

File: _summaries/bello2021lambdanetworks_1.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* How is it realized (technically)?\
+Generally, we have an query input $$(q_n, n)$$, which gets mapped to an output vector $$y_n$$ given the context $$C$$ as:\
+$$((q_n, n),C)\leftarrow y_n$$ \
+The problem here is that this function ignores the position based interaction $$(q_n, (n,m))$$. Lambda layers take into account these relation (n, m) between the query and context element positions.\
+Figure 2 shows the computational graph of the lambda layer.\
+![Bello (2021).](bello_lambda_networks_2021_1_b.PNG)
+The author also provides pseudocode to create a lambda layer.
+
+* How well does the paper perform?\
+The LambdaNetworks were tested against self-attention models, and resulted in better performance, as well as being more computationally efficient, and faster than the other models, which were sometimes slowed down and hindered by large memory costs, impractical implementations, or the requirement of huge amounts of data.\
```
Comment (SShivang) [inline _summaries/bello2021lambdanetworks_1.md:28]: You could elaborate more on the run time complexity of the lambda networks here. Why does it achieve this boost?

File: _summaries/bello2021lambdanetworks_1.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+The most impressive results are:\
+LambdaResNets achieve a 4.4x speedup over EfficientNets on ImageNet, and they are still more energy efficient.\
+In a semi-supervised learning environment, LambdaResNets achieve up to a 9.5x speedup over EfficientNet NoisyStudent checkpoints (a kind of semi-supervised learning approach).
+
```
Comment (SShivang) [inline _summaries/bello2021lambdanetworks_1.md:32]: I think the figures in the paper would be a great addition here. You could go into the different domains they explored, for example they show results on Mask-RCNN.

File: _summaries/bello2021lambdanetworks_1.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+* What interesting variants are explored?\
+The authors explore wiht hybrid architectures. They do this because even with the time and memory saving compared to self-attention, the lambda layer still incurs a quadratic time complexity. The authors study a convolution-lambda hybrid, which increases representational power at no throughput cost.
```
Comment (SShivang) [inline _summaries/bello2021lambdanetworks_1.md:35]: I believe they also explore a larger model and see how well it does. Might be an interesting add here.

**Comment by marcobueso** [comment]
Comments addressed.
Ready for final submission.

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Pull request merged**